.Lib9c.Tests/Action/SellCancellationTest.cs

```diff
@@ -64,43 +64,42 @@ public SellCancellationTest(ITestOutputHelper outputHelper)
             };
             agentState.avatarAddresses[0] = _avatarAddress;
 
-            var shopState = new ShopState();
-
             _initialState = _initialState
                 .SetState(GoldCurrencyState.Address, _goldCurrencyState.Serialize())
-                .SetState(Addresses.Shop, shopState.Serialize())
                 .SetState(_agentAddress, agentState.Serialize())
+                .SetState(Addresses.Shop, new ShopState().Serialize())
                 .SetState(_avatarAddress, avatarState.Serialize());
         }
 
         [Theory]
-        [InlineData(ItemType.Equipment, "F9168C5E-CEB2-4faa-B6BF-329BF39FA1E4", Sell.ExpiredBlockIndex, true)]
-        [InlineData(ItemType.Costume, "936DA01F-9ABD-4d9d-80C7-02AF85C822A8", Sell.ExpiredBlockIndex, true)]
-        [InlineData(ItemType.Equipment, "F9168C5E-CEB2-4faa-B6BF-329BF39FA1E4", 0, false)]
-        [InlineData(ItemType.Costume, "936DA01F-9ABD-4d9d-80C7-02AF85C822A8", 0, false)]
-        public void Execute(ItemType itemType, string guid, long requiredBlockIndex, bool contain)
+        [InlineData(ItemType.Equipment, "F9168C5E-CEB2-4faa-B6BF-329BF39FA1E4", true)]
+        [InlineData(ItemType.Costume, "936DA01F-9ABD-4d9d-80C7-02AF85C822A8", true)]
+        [InlineData(ItemType.Equipment, "F9168C5E-CEB2-4faa-B6BF-329BF39FA1E4", false)]
+        [InlineData(ItemType.Costume, "936DA01F-9ABD-4d9d-80C7-02AF85C822A8", false)]
+        public void Execute(ItemType itemType, string guid, bool contain)
         {
             var avatarState = _initialState.GetAvatarState(_avatarAddress);
             INonFungibleItem nonFungibleItem;
             Guid itemId = new Guid(guid);
+            Guid productId = itemId;
+            ItemSubType itemSubType;
+            long requiredBlockIndex = 0;
```

Comment:
This variable seems like constant variable, doesn't it? If then, how about using this with `const` keyword?